app/src/app/user/profile/profile.tpl.html

```diff
@@ -1,25 +1,25 @@
 <div layout-padding>
     <h2>User Profile</h2>
 
-    <form novalidate name="editProfileForm" ng-submit="ProfileController.updateProfile()">
+    <form novalidate name="editProfileForm" ng-submit="ProfileController.updateUser()">
         <md-content layout-padding>
             <h3>About You</h3>
             <md-whiteframe class="md-whiteframe-z1" layout="column" layout-padding>
                 <md-input-container class="md-icon-float">
                     <label for="editProfileForm.firstName">First Name</label>
                     <md-icon>account_box</md-icon>
-                    <input id="editProfileForm.firstName" ng-model="ProfileController.user.firstName" name="firstName" type="text" md-maxlength="45">
+                    <input id="editProfileForm.firstName" ng-model="ProfileController.fullUserInfo.firstName" name="firstName" type="text" md-maxlength="45">
```

Comment:
Can you switch these ids to not have the dot in them - I realised that I did it somewhere else but thinking about it now it will make jquery or css selectors fail. Just camelcase it so this is `editProfileFormFirstName`